src/JSqueeze.php

```diff
@@ -248,7 +249,19 @@ protected function extractStrings($f)
                     ++$i;
 
                     if ("\n" != $f[$i]) {
-                        isset($q[$f[$i]]) && ++$q[$f[$i]];
+                        if (isset($q[$f[$i]])) {
+                          // PHP 8.3 deprecated incrementing strings
+                          // @see https://www.php.net/manual/en/migration83.deprecated.php
+                          if (is_string($q[$f[$i]])) {
+                            // Skip these since it errors with "Argument #1 ($string) must be composed only of alphanumeric ASCII characters" 
+                            if ($q[$f[$i]] !== '\'' && $q[$f[$i]] !== '"') {
```

Comment:
Also important to note that before PHP83, `++$v` where $v is a string produced the same result (https://3v4l.org/rPnbS#v7.4.33). Therefore, we are okay to always skip `'` and `"` from being used in `str_increment`